### DISCUSSION TITLE
Maintain orientation of the Geometry Tool while rotating Canvas

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6743] [Ideas]
When you rotate the kanvas, the orientation of the Geometry tool itself doesn't change.

![image](https://github.com/opentoonz/opentoonz/assets/74098734/b5ab9a3a-2b3f-4dce-8e33-67fdaa2ffae8)

You can rotate it with the Rotate option, but I think it's more intuitive if the orientation changes automatically while you are rotating, and then if you want, use the Rotate option.

**Comment by gab3d** [comment]
...perhaps an option to do it or not, in the Tool options bar, would be the best.
"Rotate with canvas" or something like that.

**Comment by F-Burning** [comment]
Yes @gab3d, an optional function always will be better 👍

**Comment by RodneyBaker** [comment]
Tagging as UI/UX mostly just to capture the aspect of this request that focuses on user's ability to rotate the canvas but keep the tools in place.  This of course effects more than just the UI/UX but that element is what will impact the user the most as they are able to quickly and intuitively rotate the canvas/table while keeping tools in place.
